src/thor/timedep_forward.cc

```diff
@@ -97,11 +97,11 @@ bool TimeDepForward::ExpandForward(GraphReader& graphreader,
       if (trans->up()) {
         hierarchy_limits_[node.level()].up_transition_count++;
         found_valid_edge = ExpandForward(graphreader, trans->endnode(), pred, pred_idx, true,
-                                         seconds_of_week, localtime, destination, best_path) ||
+                                         localtime, seconds_of_week, destination, best_path) ||
```

Comment:
This mixup of the order of `localtime` and `seconds_of_week` seems to have been in the code for over a year?

Reply:
oops! Good catch. This code was never well tested (my only defense/excuse).

Reply:
i swear i fixed this myself a cuople months ago...